projects/collection-editor-library/src/lib/components/editor/editor.component.ts

```diff
@@ -369,11 +377,19 @@ export class EditorComponent implements OnInit, OnDestroy, AfterViewInit {
   }
 
   submitHandler() {
+    const objectType = this.editorConfig.config.objectType;
     if (!this.validateFormStatus()) {
       this.toasterService.error(_.get(this.configService, 'labelConfig.messages.error.005'));
       return;
+    } else if (objectType === 'Collection') {
```

Comment:
@vaibhavbhuva Why are you hardcoding this to `collection`.

Reply:
@pallakartheekreddy I have removed it. Please check and confirm the same